dev-db/etcd-wrapped/files/etcd-wrapper

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/bash -e
+
+# Wrapper for launching etcd via rkt.
+#
+# The version of etcd to be used can be set through ETCD_VERSION. The version
+# must be set to one of the tags published here:
+#
+#   https://github.com/coreos/etcd/releases
+
+REQUIRED_ENV_VARS="ETCD_IMG ETCD_IMG_USER ETCD_DATA_DIR"
+for rev in $REQUIRED_ENV_VARS ; do
+    if [[ -z ${!rev} ]]; then
+        echo ${rev} is not set
+        exit 1
+    fi
+done
+
+RKT=${RKT:-/usr/bin/rkt}
+NOTIFY_SOCKET=/run/systemd/notify
```

Comment:
Isn't this supposed to be set by systemd? Its pretense in the env is the signal the running process has whether it needs to notify or not.